### PR TITLE
remove the word external

### PR DESCRIPTION
**Pull request by zhengguoyong**
remove the word external,
Similar as PR: #14250

### CODE REVIEW

File: pkg/controller/service/servicecontroller.go [hunk 1]
```diff
@@ -152,17 +152,17 @@ func (s *ServiceController) init() error {
 
 	balancer, ok := s.cloud.TCPLoadBalancer()
 	if !ok {
-		return fmt.Errorf("the cloud provider does not support external TCP load balancers.")
+		return fmt.Errorf("The cloud provider does not support TCP load balancers.")
```
Comment (thockin) [inline pkg/controller/service/servicecontroller.go]: Go convention is that errors start lower-case.

**Comment by ashcrow** [comment]
This seems like a logical next step after #14250.

Travis Failure:
```
!!! 'gofmt -s' needs to be run on the following files: 
./pkg/controller/service/servicecontroller.go
```

@zhengguoyong  please update the PR after running through ```gofmt```.

**Comment by zhengguoyong** [comment]
Done.   please review,
thanks.  :)

**Comment by thockin** [comment]
LGTM.  I restarted shippable - please keep an eye on it if it fails, kick it again.

**Comment by zhengguoyong** [comment]
ok, thanks.

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Pull request merged**